locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -622,6 +624,14 @@ protected JcloudsSshMachineLocation obtainOnce(ConfigBag setup) throws NoMachine
             if (node == null)
                 throw new IllegalStateException("No nodes returned by jclouds create-nodes in " + setup.getDescription());
 
+            boolean windows = node.getOperatingSystem().getFamily().equals(OsFamily.WINDOWS);
+
+            if (windows) {
+                // FIXME: Tidy this up and allow for user-specified credentials
+                setup.put(USER, node.getCredentials().getUser());
+                setup.put(PASSWORD, node.getCredentials().getOptionalPassword().orNull());
+            }
+
             // Setup port-forwarding, if required
             Optional<HostAndPort> sshHostAndPortOverride;
```

Comment:
We should generalise `sshHostAndPortOverride`, perhaps to `connectionHostAndPortOverride`. And we should ensure it is setup for WinRM port in the windows case.

Can do that in separate PR.